src/frontend/screens/LocationInfoScreen.tsx

```diff
@@ -74,7 +74,7 @@ const InfoRow = ({label, value}: {label: string; value: string}) => (
 );
 
 export const LocationInfoScreen = () => {
-  const {location} = useLocation({maxDistanceInterval: 0});
+  const {location} = useLocation();
```

Comment:
Can't leave a comment below, but on line #94 I don't think we should display the current time if we don't have a location - it suggests to the user that there is a location

Reply:
https://github.com/digidem/comapeo-mobile/pull/387/commits/9b28ba6ce7068d30088de5ff991e378f7a915066